test/tests/CMakeLists.txt

```diff
@@ -31,7 +31,9 @@ add_orct2_test(INI IniWriterTest.cpp IniReaderTest.cpp)
 add_orct2_test(String StringTest.cpp)
 
 if(ENABLE_RCT2_TESTS)
-    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink "${CMAKE_CURRENT_LIST_DIR}/testdata" "${CMAKE_CURRENT_BINARY_DIR}/testdata")
+    if(NOT WIN32)
+        execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink "${CMAKE_CURRENT_LIST_DIR}/testdata" "${CMAKE_CURRENT_BINARY_DIR}/testdata")
```

Comment:
Why is it skipping the symlinking?

Reply:
`create_symlink` doesn't work properly on Windows, as far as I've seen. It may be sufficient to just outright copy the directory instead for Windows.